src/applications/terms-of-use/tests/MyVAHealth.unit.spec.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { render, waitFor } from '@testing-library/react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+import { $ } from 'platform/forms-system/src/js/utilities/ui';
+import MyVAHealth from '../components/MyVAHealth';
+
+const oldLocation = global.window.location;
+
+Array.from({ length: 200 }).forEach(() => {
```

Comment:
Are we running the test this many times to simulate continuous use/ strain on the route?

Reply:
Oh weird I thought I took that out. Good catch Afia